.travis.yml

```diff
@@ -24,12 +24,14 @@ before_script:
 
   # Install composer dependencies
   - composer validate
+    # for silverstripe/config tests
+  - case "$PHPUNIT_TEST" in core*) composer require --no-update mikey179/vfsStream ^1.6; esac
```

Comment:
Just require it outright, this syntax is harder to understand than Travis configuration generally is and I don't see much benefit

Reply:
Cool, I had it like that originally, but with trying to segment things I thought I should try and save those precious seconds :P
I'll revert :)